Gulpfile.js

```diff
@@ -38,14 +41,25 @@ function compile(advanced = false) {
         .pipe(sourcemaps.init())
         .pipe(closureCompiler(options))
         .pipe(sourcemaps.write('/'))
-        .pipe(gulp.dest('dist'));
+        .pipe(gulp.dest('dist/js'));
+}
+
+function cssMinify() {
+    return gulp.src('src/components/**/*.css')
+        .pipe(concatCss('css/erste.css'))
```

Comment:
You should be able to use erste.min.css here and remove the rename. This has me thinking; maybe we should drop erste.js and use erste.min.js, as it actually doesn't make a lot of difference since we now have sourcemaps. What do you say?